src/translations/indexMapper.js

```diff
@@ -315,14 +517,15 @@ class IndexMapper {
    * @param {number} amountOfIndexes Amount of inserted indexes.
    */
   insertIndexes(firstInsertedVisualIndex, amountOfIndexes) {
-    const nthVisibleIndex = this.getNotSkippedIndexes()[firstInsertedVisualIndex];
+    const nthVisibleIndex = this.getNotTrimmedIndexes()[firstInsertedVisualIndex];
     const firstInsertedPhysicalIndex = isDefined(nthVisibleIndex) ? nthVisibleIndex : this.getNumberOfIndexes();
     const insertionIndex = this.getIndexesSequence().includes(nthVisibleIndex) ? this.getIndexesSequence().indexOf(nthVisibleIndex) : this.getNumberOfIndexes();
     const insertedIndexes = arrayMap(new Array(amountOfIndexes).fill(firstInsertedPhysicalIndex), (nextIndex, stepsFromStart) => nextIndex + stepsFromStart);
 
```

Comment:
Can I ask you to split the code above to not exceeds 120 columns?

Reply:
Done: https://github.com/handsontable/handsontable/pull/6547/commits/360257b7d9eacf35885bdd918682bb0fc352028f#diff-886f922b8f171d31900bdcd191bdd61bL522-R525.